client/components/ThemeMenu.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+
+import { useGlobalContext } from "../context/GlobalContext";
+
+export default function ThemeMenu() {
+    const {
+        state: { userTheme },
+        action: { setUserTheme },
+    } = useGlobalContext();
+
+    function changeTheme(e) {
+        setUserTheme(e.target.value);
+    }
+
+    return (
+        <FormControl className="user-theme__menu">
+            <InputLabel id="user-theme__label">Theme</InputLabel>
+            <Select
```

Comment:
When it comes time to test this, take a look at the test for my bulk editor -- it's a bit tricky to test MUI select controls, but that test does it successfully. (You have to trigger a MouseUp action, if I remember correctly...)